### DISCUSSION TITLE
How to wrap macy widget around local images saved using savemedia addon and tiddlyclip

### DISCUSSION THREAD

**Discussion by arunnbabu81** [buggyj/tiddlyclip-plugin#118] [Q&A]
Most the images in my wiki are clipped using the @clipsnap rule of tiddlyclip plug in and saved automatically in a local folder using save media addon.
Images are referenced within the tiddlers in this format 

`[img[media/Tweet-DrDanMD-MOG disease 2023_4_1 23_39_40.png]]`

I want to wrap macy widget around these images as shown in [this example](https://saqimtiaz.github.io/tw5-plugins-sandbox/#Macy%3A%20Example%20with%20lightbox)

[Here](https://macy-demo.tiddlyhost.com/#Tweet-theliverdr%2FFatty%20liver) is my attempt which is not working (I am using twitter images for demo instead of local images here)
Is it possble to wrap macy widget around images of this format [img[]] as shown in the example?

**Comment by buggyj** [comment]
you need to remove the 'img' -

![image](https://user-images.githubusercontent.com/2855795/229748663-948c6d06-21f1-491c-b760-6dc27e310d9d.png)

**Reply by arunnbabu81** [reply]
It works. But I  am not sure whether using macy will cause trouble for me on long term  if the plug-in stops working. Not sure whether to use it or not. Thanks for the help @buggyj
